FFXIVClientStructs/FFXIV/Client/LayoutEngine/LayoutWorld.cs

```diff
@@ -36,4 +36,7 @@ public unsafe partial struct LayoutWorld {
 
     [MemberFunction("48 89 5C 24 ?? 48 89 74 24 ?? 57 48 81 EC ?? ?? ?? ?? 48 8B 05 ?? ?? ?? ?? 48 33 C4 48 89 84 24 ?? ?? ?? ?? 48 83 B9 ?? ?? ?? ?? ?? 48 8B F1"), GenerateStringOverloads]
     public partial byte* ResolveRsvString(byte* rsvString);
+
+    [MemberFunction("E9 ?? ?? ?? ?? CC CC CC CC CC CC CC CC CC 48 8B 11")]
```

Comment:
This sig is across function boundaries (padding between functions are the CC bytes).
It's better to sig the beginning of the function directly:
```suggestion
    [MemberFunction("40 53 55 56 57 41 54 41 55 41 56 41 57 48 81 EC ?? ?? ?? ?? 48 8B 05 ?? ?? ?? ?? 48 33 C4 48 89 84 24 ?? ?? ?? ?? 4D 8B E8")]
```

Reply:
It's a bit annoying.
You have to select the beginning of the function, create a ida pattern from selection, then search for the sig to check if it's unique and repeat that until you have the shortest possible sig without multiple results.
I think pohky does that in ghidra with a single click. 🙄

![screenshot1](https://github.com/user-attachments/assets/7c4a76e2-1abf-4b54-b14c-03311b621cd1)
![screenshot2](https://github.com/user-attachments/assets/2db1b772-788e-4ea2-9fcd-35071bb70dc1)